dizoo/classic_control/pendulum/envs/pendulum_env.py

```diff
@@ -24,6 +24,9 @@ def __init__(self, cfg: dict) -> None:
         self._reward_space = gym.spaces.Box(
             low=-1 * (3.14 * 3.14 + 0.1 * 8 * 8 + 0.001 * 2 * 2), high=0.0, shape=(1, ), dtype=np.float32
         )
+        # require discrete env
+        self._is_continue=is_continue
```

Comment:
you should modify `self._action_space` when `continuous=False`, so you can use consistent interface in other parts like `random_action` method